File: TechnologiesFundamentals/ProgrammingFundamentals/DataTypesAndVariables-Exercises/11.ConvertSpeedUnits.py
distanceInMeters = int(input())
hours = int(input())
minutes = int(input())
seconds = int(input())

hourTime = hours + minutes / 60.0 + seconds / 3600.0
kmSpeed = (distanceInMeters / 1000.0) / hourTime
metersSpeed = kmSpeed / 3.6
milesSpeed = (distanceInMeters / 1609.0) / hourTime

print("%.6f" % metersSpeed)
print("%.6f" % kmSpeed)
print("%.6f" % milesSpeed)

File: TechnologiesFundamentals/ProgrammingFundamentals/DataTypesAndVariables-Exercises/11.ConvertSpeedUnits.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int distanceM = Integer.parseInt(scanner.nextLine());
        int hours = Integer.parseInt(scanner.nextLine());
        int minutes = Integer.parseInt(scanner.nextLine());
        int seconds = Integer.parseInt(scanner.nextLine());

        int secondsConverted = (((hours * 60) * 60) + minutes * 60 + seconds);
        float mPs = ((float) distanceM / (float) secondsConverted);
        double hoursConverted = (hours + ((double) minutes / 60) + ((double) seconds / 60) / 60);
        float kmH = (((float) distanceM / 1000) / (float) hoursConverted);
        float mpH = (float) ((double) distanceM / (double) 1609) / (float) hoursConverted;

        System.out.printf("%f%n%f%n%f", mPs, kmH, mpH);
    }
}
